### ISSUE TITLE
[Proposition] For update jQuery and a solution

### ISSUE THREAD

**Issue by shawe** [themouette/jquery-week-calendar#204]
On latest versions of jQuery/jQuery UI weekCalendar stop working because jquery.browser was deprecated, if you add https://github.com/gabceb/jquery-browser-plugin like dependency, you can upgrade to latest version.

At this time, only with jquery-migrate-1.4.1.js I solved that founding the problem, I don't now if later I can have more problems, but until now, I don't needed jquery-migrate-3.0.0.js or something else to re-check any other problem.

If you are interested on upgrade, seems that you can do it without big problems.

I'm starting to update and all code that is using weekCalendar and maybe appear more news problems, are you interested in that I notify you here for solutions I found?

**Comment by n3nikita** [comment]
Yes, im interested! I had to use old versions of jQuery/jQuery UI with weekCalendar.

**Comment by shawe** [comment]
@MrBandaruk finally the customer prefer to use fullCalendar+Schedule and I don't test anything more.
